src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -301,6 +304,15 @@ private static bool IsDistroAndVersionOrHigher(Predicate<string> distroPredicate
             return false;
         }
 
+        private static bool IsQemu()
+        {
+            if (IsLinux)
+            {
+                return Environment.GetEnvironmentVariable("DOTNET_RUNNING_UNDER_QEMU") != null;
```

Comment:
What sets this env var?

Reply:
It's set manually by us if needed. Idea based on discussion: https://github.com/dotnet/runtime/pull/104094#discussion_r1665618792